packages/datadog_flutter_plugin/example/ios/Tests/DatadogRumPluginTests.swift

```diff
@@ -11,513 +11,566 @@ import XCTest
 import datadog_flutter_plugin
 
 enum ResultStatus: EquatableInTests {
-  case notCalled
```

Comment:
A lot of changes are formatting changes only. It makes things harder to review. 
Can we avoid these in the future?

Reply:
Yeah, I'm trying to slowly bring everything into 4 spaces indents. I'll do a PR with just the indentation changes to get everything in line.